test/getNumTokens.js

```diff
@@ -1,9 +1,35 @@
 /* eslint-env mocha */
-/* global contract */
+/* global contract assert */
 
-contract('PLCRVoting', () => {
+const utils = require('./utils.js');
+
+contract('PLCRVoting', (accounts) => {
   describe('Function: getNumTokens', () => {
-    it('should return the number of tokens committed by the voter for some pollID');
+    const [alice] = accounts;
+    it('should return the number of tokens committed by the voter for some pollID', async () => {
+      const plcr = await utils.getPLCRInstance();
+      const token = await utils.getERC20Token();
+      const options = utils.defaultOptions();
+      options.actor = alice;
+      options.numTokens = '40';
+
+      const startingBalance = await token.balanceOf.call(alice);
+      const pollID = await utils.startPollAndCommitVote(options);
+
+      const numTokens = await utils.as(alice, plcr.getNumTokens, alice, pollID);
+      assert.strictEqual(
+        numTokens.toString(10),
+        '40',
+        'should have 40 numTokens',
+      );
+
+      const endBalance = await token.balanceOf.call(alice);
+      assert.strictEqual(
```

Comment:
All of the state transition checks where you check this should work like this one.